uber/config.py

```diff
@@ -131,6 +143,16 @@ def __getattr__(self, name):
                 return sa.localized_now() > date_setting
         elif name.startswith('HAS_') and name.endswith('_ACCESS'):
             return getattr(c, name.split('_')[1]) in sa.AdminAccount.access_set()
+        elif name.endswith('_AVAILABLE'):
```

Comment:
You should be using ``rsplit`` here instead of ``split``.  Suppose that we have a name like ``COOL_STUFF_AVAILABLE``.  This line would result in ``item_check`` being equal to ``COOL`` instead of ``COOL_STUFF``.  By using ``rsplit`` we split from the right-hand side of the string.